sdk/keyvault/keyvault-certificates/samples/javascript/backupAndRestore.js

```diff
@@ -23,21 +19,20 @@ async function main() {
   const certificateName = "MyCertificate123129";
 
   // Creating a self-signed certificate
-  const certificate = await client.createCertificate(certificateName, {
+  const createPoller = await client.beginCreateCertificate(certificateName, {
     issuerName: "Self",
     subject: "cn=MyCert"
   });
 
-  console.log("Certificate: ", certificate);
+  const pendingCertificate = createPoller.getResult();
+  console.log("Certificate: ", pendingCertificate);
 
   const backup = await client.backupCertificate(certificateName);
 
-  // It might take less time, or more, depending on your location, internet speed and other factors.
-  await client.deleteCertificate(certificateName);
-  await delay(30000);
+  const deletePoller = await client.beginDeleteCertificate(certificateName);
+  await deletePoller.pollUntilDone();
 
   await client.purgeDeletedCertificate(certificateName);
```

Comment:
This looks suspicious. At least for me, I have to wait for purges to complete or I can't be assured they have finished.